Source/JavaScriptCore/bytecompiler/BytecodeGenerator.h

```diff
@@ -1319,6 +1318,7 @@ namespace JSC {
         RegisterID m_thisRegister;
         RegisterID m_calleeRegister;
         RegisterID* m_scopeRegister { nullptr };
+        RegisterID* m_topLevelScopeRegister { nullptr }; // This may not be set. Only Eval and Module will configure this register.
```

Comment:
If I am understanding m_topLevelScopeRegister correctly, it's used for holds a copy of scopeRegister() set at op_enter to avoid emitted repeated op_get_scope? Is that only for low tiers? So, for DFG and FTL why don't we need similar things like m_topLevelScopeRegister? Oh, maybe callFrame->callee->scope() is similar to the concept of m_topLevelScopeRegister?